dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/processor/MasterTaskPauseProcessor.java

```diff
@@ -55,9 +55,11 @@ public void process(Channel channel, Message message) {
         try (
                 LogUtils.MDCAutoClosableContext mdcAutoClosableContext = LogUtils.setWorkflowAndTaskInstanceIDMDC(
                         taskExecutionContext.getProcessInstanceId(), taskExecutionContext.getTaskInstanceId())) {
-            masterTaskExecuteRunnable.pauseTask();
-        } catch (MasterTaskExecuteException e) {
-            log.error("Pause MasterTaskExecuteRunnable failed", e);
+            try {
```

Comment:
double try? should remove it

Reply:
If we remove the first try, the close resource might be put before catch.